zerver/lib/export.py

```diff
@@ -527,6 +534,14 @@ def sanity_check_stream_data(response, config, context):
             Please investigate!
             ''')
 
+def get_user_id_filter_from_context(response, config, context):
+    # type: (TableData, Config, Context) -> Dict[str, Any]
+    user_ids = context['user_ids']
+    if user_ids:
```

Comment:
would write this with `is not None` for consistentcy with the rest of the codebase